modules/react/collection/stories/Collection.stories.mdx

```diff
@@ -126,111 +119,12 @@ cursor wraps around columns and rows when an edge of a column or row is encounte
-are used by higher level components like `Menu` and `Menu.Item` which utilize `ListBox`.
-
-#### Props
-
-<ArgsTable of={ListBox} />
-
-### ListBox.Item
-
-#### Usage
-
-The `ListBox.Item` is a simple placeholder for listbox items. The functionality of a collection item
-varies between components. For example, a `Tabs.Item` and a `Menu.Item` have shared functionality,
-but have different behavior. All collection-based components should implement a custom `Item`
-subcomponent using the collection-based behavior hooks. The [Roving Tabindex](#roving-tabindex)
-example uses the `elemPropsHook` to provide additional functionality. `elemPropsHook` is provided on
-all compound components and is useful in the example to add additional functionality without making
-a new component.
-
-#### Props
-
-<ArgsTable of={ListBox.Item} />
-
-## Models
-
-There are two supported models based on the collection system.
-
-### `useListModel`
-
-The List model contains the the state and events necessary to track items, selection, and a cursor.
-Various hooks can be used for a List model to create common behaviors associated with lists, such as
-navigating a list with a keyboard, selection (single and multiple), and virtualization.
-
-A list also has a "cursor". A cursor is often represented by focus, but it is not always a 1:1
-mapping. Think of the cursor as the focus item within the list. If the list has browser focus, the
-cursor will map to browser focus. Behaviors such as `useListRovingFocus` will map the cursor to the
-active tab stop of the list. For more information, see
-[Roving Tabindex](https://w3c.github.io/aria-practices/#kbd_roving_tabindex). `useListRovingFocus`
-adds keyboard events that map to navigation events. A [Navigation Manager](#navigation-manager) is
-used to map new cursor ids to these events. The `ListModel` takes an optional `navigation`
-configuration to change the default navigation behavior. The default navigation manager is a
-[wrappingNavigationManager](#wrappingnavigationmanager) meaning the cursor will wrap around the
-beginning and the ends. The cursor also provides a [navigationManager](#navigationmanager) that does
-not wrap. This is the default navigation for grids.
-
-The cursor also adds the concept of `orientation` which defaults to `'vertical'`. A Tab list is an
-example of a `'horizontal'` list.
-
-#### Config
-
-<ArgsTable of={ListModelConfigComponent} />
-
-#### State
-
-<ArgsTable of={ListStateComponent} />
-
-#### Events
-
-<ArgsTable of={ListEventsComponent} />
-
-### `useGridModel`
-
-The Grid model extends the List model and changes some config. For example, the `columnCount` is
-required on the grid model's configuration and `orientation` is removed.
-
-#### Config
-
-<ArgsTable of={GridModelConfigComponent} />
-
-#### State
-
-<ArgsTable of={GridStateComponent} />
-
-#### Events
-
-<ArgsTable of={GridEventsComponent} />
+<SymbolDoc name="ListBox" />
+<SymbolDoc name="useGridModel" />
```

Comment:
We have a small regression here. This stood out when reviewing the Storybook output because it has no explanation why it's here. The description is moved into the popup and is no longer on the page:

Before:
<img width="1043" alt="Screenshot 2023-02-26 at 11 06 22 PM" src="https://user-images.githubusercontent.com/1075861/221487495-e359fb97-17cd-49c0-8a7f-a7d843163690.png">

Current changes:
<img width="546" alt="Screenshot 2023-02-26 at 11 06 28 PM" src="https://user-images.githubusercontent.com/1075861/221487606-4b97cd10-c625-462e-adb5-6a30eb3574db.png">

Reply:
The description is above `useGridModel`

This happens because components have a single model, but collections offer two models. Symbols are rendered as
- description
- type

So this reads like:
## Model

{useListModel description}

`useListModel (config: ...)`

{useGridModel description}

`useGridModel (config: ...)`

Please confirm this is a problem.

Reply:
It's not, we can probably improve the language for readability but let's ship it as it is now.